src/slider.scss

```diff
@@ -6,6 +6,10 @@ $handle-desktop-dimensions: 1.25rem;
 $handle-hit-area-desktop-dimensions: 2rem;
 $handle-mobile-dimensions: 1.625rem;
 $handle-hit-area-mobile-dimensions: 2.75rem;
+$handle-outline-offset: 0.0625rem;
+$handle-outline-width: 1px;
```

Comment:
You can use `0.0625rem` here also. The `rem` using is required.

Reply:
Fixed